images/dind/node/Dockerfile

```diff
@@ -91,3 +91,5 @@ RUN systemctl enable crio-node.service
 
 # Default crio storage to vfs. overlay on overlayfs is not supported in crio.
 RUN echo "CRIO_STORAGE_OPTIONS=--storage-driver vfs" > /etc/sysconfig/crio-storage
+RUN sed -i 's/^plugin_dir .*/plugin_dir = "\/opt\/cni\/bin\/"/g' /etc/crio/crio.conf
```

Comment:
Why do we need this?

Reply:
Because crio defaults `plugin_dir` to `/usr/libexec/cni/` on Fedora and we install/expect openshift sdn plugin to be present under `/opt/cni/bin` dir.

Reply:
@pravisankar BTW when Kube 1.11 merges to Origin we can then pass --cni-bin-dir=/usr/libexec/cni to kubelet.